lib/improver/tests/utilities/cube_manipulation/test_enforce_coordinate_ordering.py

```diff
@@ -84,7 +84,7 @@ def test_move_coordinate_to_end(self):
         expected = self.cube.copy()
         expected.transpose([1, 2, 3, 0])
         result = enforce_coordinate_ordering(
-            self.cube, "realization", anchor="end")
+            self.cube, "realization", anchor_start=False)
```

Comment:
There aren't any existing tests to check what happens to any aux coordinates associated with a dim coordinate. I think it your code does the right thing in this case, but it might be worth adding a test to cover this. What do you think?

Reply:
Hmm good point.  I'd say it's the responsibility of `cube.transpose()` to deal with that, but there's no harm in checking it does!  I'll write a test.

Reply:
Thank you